leshan-server-cluster/src/main/java/org/eclipse/leshan/server/cluster/serialization/RegistrationUpdateSerDes.java

```diff
@@ -126,6 +134,12 @@ public static RegistrationUpdate deserialize(byte[] data) throws UnknownHostExce
             }
         }
 
-        return new RegistrationUpdate(regId, identity, lifetime, sms, b, linkObjs);
+        Map<String, String> addAttr = new HashMap<>();
+        JsonObject o = (JsonObject) v.get("addAttr");
+        for (String k : o.names()) {
+            addAttr.put(k, o.getString(k, ""));
```

Comment:
you are not retrieving the serialized values, just the keys.

Reply:
Are you sure? I think it uses the method `put(key,value)` parsing the string to get the value. I copied this code from the `RegistrationSerDes`.

Reply:
you are right, thanks.